### PR TITLE
3T tools updated

### PR DESCRIPTION
**Pull request by tomzahn**

### CODE REVIEW

File: source/tools/administration-interfaces.txt [hunk 1]
```diff
@@ -414,16 +414,16 @@ Third-Party Commercial Tools
 
 - <http://3t.io/data-compare/>
 
-3T Data Compare is a multi-platform data comparison tool for MongoDB,
-and includes the following features:
+3T Data Compare is a MongoDB data comparison tool for ``Mac``, ``Windows``, and ``Linux``.
```
Comment (mpaik) [inline source/tools/administration-interfaces.txt]: The phrase 'is a MongoDB data comparison tool' is not acceptable as it implies that MongoDB is the source; please find alternate phrasing.

File: source/tools/administration-interfaces.txt [hunk 2]
```diff
@@ -414,16 +414,16 @@ Third-Party Commercial Tools
...
 
-- Graphical side-by-side comparisons make it straightforward to understand the differences in your data.
-- Easily visualise comparison results to help identify how critical data differs across environments.
-- Highlight what new data and changes are required between different deployments of your systems.
-- Examine which changes have occurred between current databases and previous backups.
-- Quickly solve data-related setup problems in new deployments.
-- Easily diagnose configuration problems in development, testing and production systems.
-- Multi-platform: Mac OS X, Windows, Linux.
+- Graphical side-by-side comparisons make it straightforward to understand the differences in your data
+- Easily visualise comparison results to help identify how critical data differs across environments
```
Comment (mpaik) [inline source/tools/administration-interfaces.txt]: s/visualise/visualize/ - All MongoDB documentation should be in US English.

File: source/tools/administration-interfaces.txt [hunk 3]
```diff
@@ -432,17 +432,21 @@ and includes the following features:
...
-
-- Edit any MongoDB data type directly in a tree-view, table-view or in a lossless JSON editor.
-- Edit complex structures such as embedded documents or arrays-in-arrays-in-embedded-documents in both a tree-view or table-view effortlessly.
-- See live evaluation of your data as you type (very handy for data types such as Date, Timestamp, etc).
-- Seamlessly step into embedded objects, either for all documents simultaneously or for just individual documents.
-- Build JSON queries with simple drag-and-drop.
-- Copy & paste documents across databases and collections.
-- Import & export documents from and to different databases, collections, and JSON files.
-- Multi-platform: Mac OS X, Windows, Linux.
+3T MongoChef is a MongoDB GUI for ``Mac``, ``Windows``, and ``Linux``.
```
Comment (mpaik) [inline source/tools/administration-interfaces.txt]: As above, 'MongoDB GUI' leads to possible misattribution. Please fix.

File: source/tools/administration-interfaces.txt [hunk 4]
```diff
@@ -432,17 +432,21 @@ and includes the following features:
...
+Full support for ``MongoDB 2.4``, ``MongoDB 2.6``, and ``MongoDB 3.0``.
+
+- ``In-place editing`` inside your tree-view, table-view or in a lossless JSON editor
```
Comment (mpaik) [inline source/tools/administration-interfaces.txt]: Double backticks here and elsewhere will be interpreted as casting to fixed-width font in RST - see http://docutils.sourceforge.net/docs/user/rst/quickref.html

File: source/tools/administration-interfaces.txt [hunk 5]
```diff
@@ -432,17 +432,21 @@ and includes the following features:
...
+- See live evaluation of your data as you type (very handy for data types such as Date, Timestamp, etc)
+- Seamlessly step into embedded objects, either for all documents simultaneously or for just individual documents
+- Visual ``query builder`` to easily build even the most complex MongoDB queries
+- Visual ``index management``
+- Visual ``user & role management``
+- ``GridFS`` with drag-and-drop
+- Full-blown ``power shell`` with superb ``auto-completion``
+- ``Copy & paste collections`` and  documents across servers and databases
+- ``Import & export`` documents from and to different databases, collections, JSON, and CSV files
+- actively maintained with frequent updates
```
Comment (mpaik) [inline source/tools/administration-interfaces.txt]: s/actively/Actively/

File: source/tools/administration-interfaces.txt [hunk 6]
```diff
@@ -451,17 +455,19 @@ and includes the following features:
 
 - <http://3t.io/schema-explorer/>
 
-With the multi-platform 3T Schema Explorer, it's easy to explore the schema of your MongoDB data:
+3T Schema Explorer is a MongoDB ``schema analysis`` and ``documentation`` GUI tool for ``Mac``, ``Windows``, and ``Linux``.
```
Comment (mpaik) [inline source/tools/administration-interfaces.txt]: Attribution.

**Comment by mpaik** [comment]
Bulk closing stale pull requests.

**Comment by tomzahn** [comment]
Thanks for your comments!
I have incorporated your remarks and issued a new pull request (#273).

**Pull request closed without merging**